user/pages/03.commerce2/02.developer-guide/04.customers/02.addresses/01.address-formats/docs.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Address Formats
+taxonomy:
+    category: docs
+---
+
+The *Address* module provides a custom *address* field that can have the following properties:
+
+- Given name (First name)
+- Additional name (Middle name / Patronymic)
+- Family name (Last name)
+- Organization
+- Address line 1
+- Address line 2
+- Postal code
+- Sorting code
+- Dependent Locality (Neighborhood / Suburb)
+- Locality (City)
+- Administrative area (State / Province)
+- Country (limited to pre-defined list of *available* countries)
+
+### What is an *address format*?
+Address formats are defined on a per-country basis. They make it possible to dynamically alter the form used to enter addresses based on the selected country. Without address formats, an address form might look something like this:
```

Comment:
I'd kind of expect the first sentence here to actually provide the definition; something like, "An address format is the definition of how a country's addresses should be rendered for display and how the form for entering an address for that country should be rendered and validated."